pepys_admin/maintenance/widgets/filter_widget_utils.py

```diff
@@ -122,7 +123,7 @@ def handle_three_variables(class_obj, output, idx, query_list, data_store):
 
     if ops == "LIKE":
         try:
-            if isinstance(col.type, UUIDType):
+            if isinstance(col.type, UUIDType) or isinstance(col.type, UUID):
```

Comment:
Could you explain what the issue was here, please @robintw ?

Reply:
I mentioned it briefly in the Developer Notes in the PR description, but basically this if statement is checking whether the column is a UUID. If it is, then it casts it to string before trying to do a `LIKE` operation on it. However, we were only checking whether it was a `UUIDType` (which is the type used for UUIDs on SQLite), and not checking against `UUID` (which is the type used for UUIDs on Postgres). Therefore, it failed on Postgres but worked on SQLite.

Reply:
Aah, got you - thanks.